test/server/models/Update.test.js

```diff
@@ -195,16 +198,60 @@ describe('server/models/Update', () => {
         const usersToNotify = await update.getUsersToNotify();
         const receivedEmails = usersToNotify.map(u => u.email);
 
-        const expectAllEmailsFrom = usersList => {
-          return usersList.forEach(user => expect(receivedEmails).to.include(user.email));
-        };
-
-        expectAllEmailsFrom(parentCollectiveAdmins);
-        expectAllEmailsFrom(collectiveAdmins);
-        expectAllEmailsFrom(individualBackersUsers);
-        expectAllEmailsFrom(getOrganizationAdminUsers());
+        expectAllEmailsFrom(parentCollectiveAdmins, receivedEmails);
+        expectAllEmailsFrom(collectiveAdmins, receivedEmails, receivedEmails);
+        expectAllEmailsFrom(individualBackersUsers, receivedEmails);
+        expectAllEmailsFrom(getOrganizationAdminUsers(), receivedEmails);
         expect(usersToNotify.length).to.eq(expectedPrivateTotal);
       });
+
+      it('Notifies child collective users when parent collective public update is made', async () => {
+        collective.remove;
```

Comment:
What is this doing?

Reply:
Ops, seems like a mistakenly added part. I've removed it. Thanks.